let u = undefined;

try {
    console.log(u);
    // u.x = 12
} catch (e) {
    console.log('Błąd');
}
